From 4d8016093c1b56bee9682ccb206bf210995a6cef Mon Sep 17 00:00:00 2001
From: Mark Enriquez <enriquezmark36@gmail.com>
Date: Sun, 15 Jul 2018 16:11:11 +0800
Subject: [PATCH 08/15] hwcomposer: scx15: Remove FramebufferNativeWindow.h
 include

It's already obsolete and has been deleted from source causing compilation errors to occur.

Change-Id: I9e6de9cf294791da1a003f17f249f363d435ee4d
Signed-off-by: Mark Enriquez <enriquezmark36@gmail.com>
---
 hwcomposer/scx15/OverlayComposer/OverlayNativeWindow.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/hwcomposer/scx15/OverlayComposer/OverlayNativeWindow.h b/hwcomposer/scx15/OverlayComposer/OverlayNativeWindow.h
index fe3e72c..1f8727d 100644
--- a/hwcomposer/scx15/OverlayComposer/OverlayNativeWindow.h
+++ b/hwcomposer/scx15/OverlayComposer/OverlayNativeWindow.h
@@ -33,7 +33,6 @@
 #ifndef _OVERLAY_NATIVEWINDOW_H_
 #define _OVERLAY_NATIVEWINDOW_H_
 
-#include <ui/FramebufferNativeWindow.h>
 #include <ui/ANativeObjectBase.h>
 #include <ui/Fence.h>
 #include "gralloc_priv.h"
-- 
2.23.0

